SU2_CFD/src/iteration_structure.cpp

```diff
@@ -2794,165 +2850,33 @@ void CDiscAdjFEAIteration::Iterate(COutput *output,
                                         unsigned short val_iInst) {
 
 
-  unsigned long IntIter = 0, nIntIter = 1;
   bool dynamic = (config_container[val_iZone]->GetDynamic_Analysis() == DYNAMIC);
 
-  config_container[val_iZone]->SetIntIter(IntIter);
-
-  nIntIter = config_container[val_iZone]->GetDyn_nIntIter();
-
-  for(IntIter = 0; IntIter < nIntIter; IntIter++){
-
-    /*--- Set the internal iteration ---*/
-
-    config_container[val_iZone]->SetIntIter(IntIter);
-
-    /*--- Set the adjoint values of the flow and objective function ---*/
+  unsigned long nIntIter = config_container[val_iZone]->GetnIter();
```

Comment:
I just noticed GetnIter() returns the value of the variable Iter and not nIter (which does not exist) from the CConfig class. 
It would be nice to have the S/Get<VarName> right? 
More a general question than a change request.